src/components/Task/Difficulty/DifficultyLabel/DifficultyLabel.tsx

```diff
@@ -6,67 +6,52 @@ import { DifficultyEnum } from '../../../../models/difficulty.enum';
 
 interface DifficultyLabelProps {
     difficulty: DifficultyEnum;
+    active?: boolean;
```

Comment:
the is no active. should be selected instead.

Reply:
ok